examples/model_building/plot_hyperparameter_tuning_with_scikit-learn.py

```diff
@@ -348,7 +348,7 @@
 #
 best_run = search_results[search_results['rank_test_score'] == 1].squeeze()
 print(
-    f"Best hyperparameters were {best_run['params']} which gave a validation "
+    f"The best hyperparameters gave a validation "
```

Comment:
why do we modify this? if it's intentional we you should remove the `f-string` in front